apps/contracts/contracts/Zengo.sol

```diff
@@ -77,24 +97,35 @@ contract ZengoDAO is PermissionsEnumerable, ContractMetadata {
 
     function submitProposal(
         string calldata _title,
-        string calldata _description
+        string calldata _proposalDescription,
+        string calldata _evidenceDescription,
+        uint256 _latitude,
+        uint256 _longitude,
+        string _proposalType,
+        string _streetAddress
```

Comment:
these are the required fields to submit a proposal?

```
title
proposalType
proposalDescription
evidenceDate
evidenceDescription
evidenceIpfsUrl // the evidence file is beeing uploaded to IPFS and we need to to store the url, right?
streetAddress
latitude
longitude
 ```
   
  just asking :smile: 
  Hope it helps

Reply:
Yes